src/pages/help-centre/_article.js

```diff
@@ -23,6 +23,7 @@ const ContactContainer = styled.div`
 export const Article = ({ children, header }) => {
     return (
         <Layout>
+            <SEO title={localize('Help Center | Account')} />
```

Comment:
```suggestion
            <SEO title={localize(`Help Center | ${header}`)} />
```

Reply:
Ah, that was test :D Sorry. Forgot to change.